v3/integration/lints/go.mod

```
@@ -0,0 +1,3 @@
+module github.com/zmap/zlint/v3/integration/lints
+
+go 1.14
```

If we keep the submodule it should use the same go version as `go.mod` from the root of `v3`:

```suggestion
go 1.15
```